app/publisher/publisher.go

```diff
@@ -18,18 +19,39 @@ type Interface interface {
 }
 
 // Stdout implements publisher.Interface and sends to stdout
-type Stdout struct{}
+type Stdout struct{
+	ExcludeList []string
+}
+
+// CheckExclusionList checks the exclusion list for matches
+func CheckExclusionList(excludes []string, msg string) bool {
+	for _, value := range excludes {
+		if len(value) > 0 && !strings.HasPrefix(value, "#") {
+			if match, _ := regexp.MatchString(strings.ToLower(value), strings.ToLower(msg)); match {
```

Comment:
ignoring errors generally makes me worry. In this case, I see no reason to suppress it. In rare cases, it makes sense but documenting the reason (in a comment)  will help to follow the logic.

Reply:
Okay it's logging errors from regexp.MatchString now